BasicRdl/BasicRdl.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{BEDCC5C5-83B2-41ED-AC9F-895757553BF2}</ProjectGuid>
+    <MinIMEVersion>6.0.0.4</MinIMEVersion>
```

Comment:
I think that the fourth number is the release candidate number, so I think that we should use max 3 numbers in the MinIMEVersion element. So 6.0.0 would be the appropriate version here. The logic in the API should then ignore everything after the 3rd number.